theme6/source/stylesheets/site.css.scss

```diff
@@ -1,24 +1,159 @@
-body {
-  background-color: #fbc547;
-  color: #333;
-  font-family: -apple-system, BlinkMacSystemFont, "Avenir Next", "Avenir",
-    "Segoe UI", "Lucida Grande", "Helvetica Neue", "Helvetica", "Fira Sans",
-    "Roboto", "Noto", "Droid Sans", "Cantarell", "Oxygen", "Ubuntu",
-    "Franklin Gothic Medium", "Century Gothic", "Liberation Sans", sans-serif;
-  padding: 18vh 1rem;
-  text-align: center;
-}
-
-a {
-  color: rgba(#000, 0.7);
-
-  &:focus,
-  &:hover {
-    color: rgba(#000, 0.6);
-  }
-}
-
-.middleman-logo {
-  margin-bottom: 1rem;
-  width: 10rem;
-}
+
+
+/*==========Stylesheet DESKTOP========*/
+    footer {
+      bottom: 0;
+      display: inline;
+      width: 100%;
+      
+    }
+
+    .information{justify-content: center;}
+
+    li{display: inline;}
+
+    .socialmedia{
+        color: #72808C;
+        font-family:'Montserrat', sans-serif;
+        font-size: 18px;
+        line-height: 21px;
+    }
+
+    .sm {
+        height: 35px;
+        width: 93px;
+        border: 2px solid #72808C;
+        border-radius: 3px;
+    }
+    #fb{
+      background: url("../images/fb.svg");
+      background-position: center center;
+      background-repeat: no-repeat;
+      margin-right: 23px;
+    }
+    #twt{
+        background: url("../images/twt.svg");
+        background-position: center center;
+        background-repeat: no-repeat;}
+    #g{
+        background: url("../images/g+.svg");
+        background-position: center center;
+        background-repeat: no-repeat;
+        margin-left: 23px;
+      }
+
+    .newsletter {
+      color: #72808C;
+      font-family: 'Montserrat', sans-serif;
+      font-size: 18px;
+      line-height: 21px;
+      text-align: center;
+    }
+
+    footer .box-social {
+      align-content: center;
+      width: 350px;
+    }
+
+    footer .box-social button:hover {
+      cursor: pointer;
+    }
+
+    footer .contenedorsocial {
+      display: inline-flex;
+      justify-content: center;
+      align-items: center;
+      width: 66%;
+    }
+
+    footer .contenedorsocial .logo {
+      margin-left: 96px;
+      margin-right: 96px;
+    }
+
+    footer .box-social p{
+      text-align: center;
+    }
+    footer .contenedornews{
+      width: 33%;
```

Comment:
And another one here, can you do this another way than adding percentages please.